### PR TITLE
Backport control-related changes from master to jitaas

### PR DESCRIPTION
**Pull request by harryyu1994**
Signed-off-by: Harry Yu <harryyu1994@gmail.com>

### CODE REVIEW

**Comment by harryyu1994** [comment]
@mpirvu another backporting item ready for review.

File: runtime/compiler/control/HookedByTheJit.cpp [hunk 1]
```diff
@@ -7104,6 +7150,13 @@ int32_t setUpHooks(J9JavaVM * javaVM, J9JITConfig * jitConfig, TR_FrontEnd * vm)
          return -1;
          }
       }
+
+   // Give the JIT a chance to do stuff after the VM is initialized
```
Comment (mpirvu) [inline runtime/compiler/control/HookedByTheJit.cpp]: This code seems to be a duplicate of the one above it. It needs to be removed.

**Comment by mpirvu** [comment]
Jenkins test sanity.functional+jitaas xlinux jdk8,jdk11 depends eclipse/openj9-omr#jitaas

**Comment by mpirvu** [comment]
Only the known `cmdLineTester_gcsuballoctests_0` test failed, hence merging.

**Pull request merged**